scripts/travis-test.sh

```diff
@@ -18,17 +18,19 @@
 set -o nounset
 set -o errexit
 
+RTN=0
 SCRIPT_DIR="$(cd "$(dirname "$0")"; pwd -P)"
 source "${SCRIPT_DIR}/travis-common.sh"
 
 log_and_run() {
   echo $*
-  exec $*
+  (exec $*) && true
```

Comment:
My understanding is that this runs the command in a subshell and captures the exit code, even though this script is running in error exit mode. That way we can keep running tests even if one of them fails.